Release

matentzn opened the pull request:
Hey, first draft of fly-stage.obo! Check also the other obo artefacts just to be safe :P

Clare72 commented on fly-stage.obo:
```diff
@@ -0,0 +1,2038 @@
+format-version: 1.2
+data-version: fbdv/releases/2019-08-08/fbdv-simple.owl
+date: 08:08:2019 12:42
+default-namespace: FlyBase_development_CV
```
Can we change this to:
>default-namespace: FlyBase development CV

just in this (FlyBase) version, looks like it has always had underscores in -simple

matentzn replied:
Dont you prefer to just do this for everything? editing this in the edit file instead?

matentzn replied:
I fixed it now for Flybase, but you might wanna edit the edit file anyways.

Clare72 commented on fly-stage.obo:
```diff
@@ -0,0 +1,2038 @@
+synonym: "pupal stage P15(i)" EXACT []
+is_a: FBdv:00005259 ! developmental stage
+relationship: immediately_preceded_by FBdv:00005365 ! pharate adult stage P14
+relationship: part_of FBdv:00005366 ! pharate
```
This one should also be 'xref_analog' as on line 2001

matentzn replied:
Which one? I think the snippet points on the wrong spot.

Clare72 replied:
line 2035:
>xref: OBO_REL:preceded_by

snippets are mostly wrong, it is in the right place for me looking at the diff.

matentzn replied:
Fixed.

Clare72 commented on fly-stage.obo:
```diff
@@ -0,0 +1,2038 @@
+synonym: "pupal stage P15(i)" EXACT []
+is_a: FBdv:00005259 ! developmental stage
+relationship: immediately_preceded_by FBdv:00005365 ! pharate adult stage P14
+relationship: part_of FBdv:00005366 ! pharate
```
Might be best to get rid of double comments, like you do with names and definitions (for all files, not just FlyBase), not sure how much this matters.

matentzn replied:
Yeap, thanks. Done.

Clare72 commented on fly-stage.obo:
```diff
@@ -0,0 +1,2038 @@
+synonym: "pupal stage P15(i)" EXACT []
+is_a: FBdv:00005259 ! developmental stage
+relationship: immediately_preceded_by FBdv:00005365 ! pharate adult stage P14
+relationship: part_of FBdv:00005366 ! pharate
```
please add
>namespace: relationship

matentzn replied:
Can you remind me why? This shot not be there at all in my opinion.. Sorry and thanks.

Clare72 replied:
It is in the SOP for preparing the flybase version:
https://github.com/VirtualFlyBrain/Management/blob/master/doc/preparing_FB_ontologies_for_release_for.md

Other than that I don't know why either.

matentzn replied:
Done

Clare72 commented on fly-stage.obo:
```diff
@@ -0,0 +1,2038 @@
+synonym: "pupal stage P15(i)" EXACT []
+is_a: FBdv:00005259 ! developmental stage
+relationship: immediately_preceded_by FBdv:00005365 ! pharate adult stage P14
+relationship: part_of FBdv:00005366 ! pharate
```
please add
>namespace: relationship

Clare72 commented on fly-stage.obo:
```diff
@@ -0,0 +1,2038 @@
+synonym: "pupal stage P15(i)" EXACT []
+is_a: FBdv:00005259 ! developmental stage
+relationship: immediately_preceded_by FBdv:00005365 ! pharate adult stage P14
+relationship: part_of FBdv:00005366 ! pharate
```
>namespace: external

needs to be stripped out from here and probably also from happens_during and any other typedefs it might pop up on in future.

matentzn replied:
Done

matentzn commented:
Ok in the latest [commit](https://github.com/FlyBase/drosophila-developmental-ontology/pull/10/commits/6228d047b260a0c32600dbc365cdf95dbfb6dedc), I fixed:

- removed namespace:external
- removed double comments
- changed default namespace

matentzn commented:
Waiting for your comments for the rest.
